creator-node/src/app.ts

```diff
@@ -1,12 +1,14 @@
+/* eslint-disable import/first */
 import type { Response, NextFunction, IRoute } from 'express'
-import type { CustomRequest } from './utils'
+import type { CustomRequest } from './utils/types'
 
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import prometheusMiddleware from 'express-prom-bundle'
 import _ from 'lodash'
 
+console.log('Starting app.js')
```

Comment:
I actually wanted to keep it because the app is so slow to start up that it's helpful to know it actually reached this file. it's just a one-time log so no spam to worry about